src/printer-postcss.js

```diff
@@ -345,7 +345,10 @@ function genericPrint(path, options, print) {
           indent(
             concat([
               softline,
-              join(concat([",", line]), path.map(print, "groups"))
+              join(
+                concat([",", parent.type === "value-value" ? hardline : line]),
+                path.map(print, "groups")
```

Comment:
As you've probably noticed, the AST nodes in CSS/SCSS/Less are super vague and super general. While this PR looks good, I'm worried about what unintended side effects this might/will have. Our tests aren't very comprehensive. But I guess we could also just release it and do the regular couple of point releases afterwards with CSS regression fixes ¯\\\_(ツ)\_/¯

Reply:
Would it help if we checked `startsWith('$')` on the parent value? If that doesn't give any more assurance, I am fine fielding any regressions that arise related to this.